docs/content/preview/develop/change-data-capture/using-logical-replication/yugabytedb-connector-properties.md

```diff
@@ -696,6 +696,73 @@ The type of LSN to use for the specified replication slot:
 * SEQUENCE - A monotonic increasing number that determines the record in global order within the context of a slot.
 * HYBRID_TIME - A hybrid time value that can be used to compare transactions across slots.
 
+##### streaming.mode
+
+Specified whether the connector should use a single task mode to stream changes or multi task mode.
+
+* `default` uses the single task mode and streams all the changes using it.
+* `parallel` uses a multi task mode and streams changes using the number of specified replication slots.
```

Comment:
You specify the slots to use using the slot.names, publication.names, or slot.ranges property.